### PR TITLE
Update customer-key-set-up.md

### PR DESCRIPTION
**Pull request by sathana87**

### CODE REVIEW

File: microsoft-365/compliance/customer-key-set-up.md [hunk 1]
```diff
@@ -153,6 +153,17 @@ Before contacting the Microsoft 365 team, you must do the following steps for ea
 ### Create a premium Azure Key Vault in each subscription
 
 The steps to create a key vault are documented in [Getting Started with Azure Key Vault](/azure/key-vault/general/overview), which guides you through installing and launching Azure PowerShell, connecting to your Azure subscription, creating a resource group, and creating a key vault in that resource group.
+
+SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:157]:
```suggestion
SharePoint Online uses Azure Key Vaults where you store the keys that you want to use to encrypt your documents and files in SharePoint Online and OneDrive for Business. 
```
```suggestion
SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:157]:
```suggestion
SharePoint Online uses Azure Key Vaults where you store the keys that you want to use to encrypt your documents and files in SharePoint Online and OneDrive for Business. 
```
```suggestion
SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 
```

File: microsoft-365/compliance/customer-key-set-up.md [hunk 2]
```diff
@@ -153,6 +153,17 @@ Before contacting the Microsoft 365 team, you must do the following steps for ea
...
+SharePoint Online (SPO) uses Azure Key Vaults where customer store the keys that they want use to encrypt their documents and files in SharePoint Online and OneDrive for Business. 
+
+ Azure Key Vault provides two types of containers, 
+
+
+1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   
+
+2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 
+
+ SPO supports both Standard/Premium vaults as well as Managed HSM.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:166]:
```suggestion
 Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.
```
```suggestion
 Azure Key Vault provides two types of containers, 


1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SPO supports both Standard/Premium vaults as well as Managed HSM.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:166]:
```suggestion
 Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.
```
```suggestion
 Azure Key Vault provides two types of containers, 


1) Premium key Vaults for storing and managing cryptographic keys, secrets, certificates and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

2) Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2. 

 SPO supports both Standard/Premium vaults as well as Managed HSM.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:166]:
```suggestion
  Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2.

SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:166]:
```suggestion
  Azure Key Vault provides two types of containers:

- Premium Key Vaults for storing and managing cryptographic keys, secrets, certificates, and storage account keys. These come in Standard and Premium vaults. These vaults are certified to FIPS 140-2 Level-2.   

- Managed HSM pool for storing and managing HSM-backed cryptographic keys. Managed HSMs are certified to FIPS 140-2 Level-2.

SharePoint Online supports both Standard and Premium vaults as well as Managed HSM.```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sathana87, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes. Thanks

File: microsoft-365/compliance/customer-key-set-up.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md:157]:
```suggestion
SharePoint Online uses Azure Key Vaults where you store the keys that you want to use to encrypt your documents and files in SharePoint Online and OneDrive for Business. 
```

**Comment by yogkumgit** [comment]
@sathana87 Please update . Thanks

**Comment by yogkumgit** [comment]
@sathana87 Gentle reminder.

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes, in the event of no response we may may have to close this PR. Thanks

**Comment by yogkumgit** [comment]
@sathana87 We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
Please feel free to re-open this issue if you see an improvement is required in the specific area of this documentation. We will certainly update this documentation with that valuable information. Thanks again, for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Pull request closed without merging**